package.json

```diff
@@ -25,39 +25,41 @@
   "license": "Apache-2.0",
   "bugs": {},
   "homepage": "https://github.com/strimzi/strimzi-ui#readme",
-  "dependencies": {
+  "dependencies": {},
+  "devDependencies": {
     "@babel/core": "^7.10.5",
     "@babel/preset-env": "^7.10.4",
-    "babel-loader": "^8.1.0",
-    "compression-webpack-plugin": "^4.0.0",
-    "html-webpack-plugin": "^4.3.0",
-    "mini-css-extract-plugin": "^0.9.0",
-    "optimize-css-assets-webpack-plugin": "^5.0.3",
-    "sass": "^1.26.10",
-    "terser-webpack-plugin": "^3.0.7",
-    "webpack": "^4.43.0"
-  },
-  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^4.2.0",
```

Comment:
Just noticed with the eslint plugins - no changes have been made for the eslint config to make use of these?